R/tlf_ae_summary.R

```diff
@@ -58,12 +58,13 @@ tlf_ae_summary <- function(outdata,
 
   # Title
   # Define title
-  if (is.null(title)) {
+  if ("analysis" %in% title | "observation" %in% title | "population" %in% title) {
     title <- collect_title(outdata$meta,
       outdata$population,
       outdata$observation,
       parameters[1],
-      analysis = "ae_summary"
+      analysis = "ae_summary",
+      title_order = title
```

Comment:
in the case where this `if` statement is satisfied, do you expect always all three "analysis", "observation" and "population" to be in the title vector?
It seems like you want something like `if(all(c("analysis", "observation", "population") %in% title))`

Reply:
Hi @BrianLang . Sometimes we don't need all these three component for title. For example, we only need "analysis" and "population". That is the main reason I want to update. 

If user really want have a title like `title = c("analysis", "this is a real title", "this is another real title")`, they could input `"analysis \n  this is a real title \n this is another real title"` instead. Maybe we could have more document for this issue to tell user "analysis", "observation", "population" are the special keyword we use? Also welcome to any other idea for the check the `title` argument input!

Reply:
I also thought about adding one more argument, such as `title_order` to control the title component. But I feel there might make user confused about the functionality of `title` and `title_order`.